Makefile

```diff
@@ -3,7 +3,7 @@
 all: format lint test
 
 dev:  ## Install a development environment
-	python3 -m pip install --user --upgrade .[dev]
+	python3 -m pip install --user --upgrade -e .[dev]
```

Comment:
typo?

Reply:
Actually, no. The idea here is to “install” a development version of kalamine, i.e. just a link to the working directory. That’s the way it used to work and I think it might have been accidentally destroyed during the recent CI additions. 

At the moment I still have to uninstall / reinstall kalamine to make it work (e.g. to display the proper version number). I hope this will fix it.